update google screenshots (#2388)
wso2/docs-is: en/docs/learn/configuring-google.md

# Configuring Google

Google can be used as a federated authenticator in WSO2 Identity
Server. Follow the instructions given below to configure WSO2 Identity Server to authenticate users using their Google user accounts.

1.  To navigate to the federated authenticators configuration section, follow the instructions given below.
    1.  Sign in. Enter your username and password to log on to the [Management Console](../../setup/getting-started-with-the-management-console).
        
    2.  Navigate to the **Main** menu to access the **Identity** menu. Click **Add** under **Identity Providers**. For more information, see [Adding and Configuring an Identity Provider](../../learn/adding-and-configuring-an-identity-provider).
        
    3.  Fill in the details in the **Basic Information** section. 

> 2.  Register OAuth 2.0 Application in Google. As the first step, go to [Google API Console](https://console.developers.google.com)

Reviewer: ```suggestion
2.  Register OAuth 2.0 Application in Google. First, go to [Google API Console](https://console.developers.google.com)
```

    and navigate to the **Credentials** tab from the sidebar. 
    

> 3.  Click **Create credentials** in the top navigation.

Reviewer: ```suggestion
3.  Click **Create credentials** on the top navigation.
```

    ![create-credentials](../assets/img/learn/create-credentials.png)

> 4.  You can configure the OAuth web application in Google by selecting **OAuth Client ID**. You can find more details [here](https://developers.google.com/identity/protocols/OpenIDConnect).

Reviewer: ```suggestion
4.  Select **OAuth Client ID** to configure the OAuth web application in Google. For more information, see the [OpenID Connect] (https://developers.google.com/identity/protocols/OpenIDConnect)topic in the Google Developer documentation.
```

      
    ![register-oauth2](../assets/img/learn/oauth-clientid.png)

5.  Select **Web application** as the **Application type** in the dropdown. 

    ![choose-application-type](../assets/img/learn/choose-application-type.png)
     
6.  Enter a name (e.g., SampleWebApplication). 
      

> 7.  Add a redirect URI, `https://localhost:9443/commonauth` by clicking **ADD URI** within **Authorized redirect URIs** (this is the endpoint in WSO2 Identity Server that accepts the response sent by Google).  

Reviewer: ```suggestion
7.  Click **ADD URI** within **Authorized redirect URIs** and add `https://localhost:9443/commonauth` as a redirect URI (this is the WSO2 Identity Server endpoint that accepts the response sent by Google).  
```

    ![redirect-uris](../assets/img/learn/redirect-uris.png)

> 8.  Click **Create**. Make note of the **Client ID** and **Client Secret** which appears in the pop-up. 

Reviewer: ```suggestion
8.  Click **Create**. Make note of the **Client ID** and **Client Secret** that appears in the pop-up. 
```
